lib/report/controller.rb

```diff
@@ -348,8 +348,9 @@ def available_values
   # @param query An optional query added to the disjunction qualifiying reports to be returned.
   def find_optional_report(query = '1=0')
     if params[:id]
-      @query = report_engine.find(params[:id].to_i,
-                                  conditions: ["#{is_public_sql} OR (#{user_key} = ?) OR (#{query})", current_user.id])
+      @query = report_engine
+               .where(["#{is_public_sql} OR (#{user_key} = ?) OR (#{query})", current_user.id])
+               .find(params[:id].to_i)
```

Comment:
Only in the master branch right now but I am really looking for the ability to write `or` statements: https://github.com/rails/rails/commit/9e42cf019f2417473e7dcbfcb885709fa2709f89